### PR TITLE
TPC: Adding PID from tracking to skimmed data

### PR DESCRIPTION
**Pull request by matthias-kleiner**

### CODE REVIEW

**Comment by miranov25** [comment]
## Usage of PID in the reconstruction -monitoring and performance parameterization

This modification to the skimming code is crucial for assessing how mass hypotheses affect both the quality of tracks and the efficiency of track matching. Furthermore, it will be instrumental in correcting and parameterizing performance based on PID accuracy and misidentification rates.

File: Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx [hunk 1]
```diff
@@ -1342,6 +1342,7 @@ class TPCTimeSeries : public Task
                             << "nClITS=" << nClITS
                             << "chi2ITS=" << chi2ITS
                             << "chi2match_ITSTPC=" << chi2match_ITSTPC
+                            << "PID=" << trkOrig.getPID()
```
Comment (shahor02) [inline Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx]: Note that like that you are saving the PID as a class. If you want to store just the integer, use `trkOrig.getPID().getID()`.

Comment (matthias-kleiner) [reply Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx]: Does it matter for the output file size?

@miranov25 do you prefer to store the class or just the ID?

**Comment by miranov25** [comment]
Storing integer will be indeed enough

File: Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx (hunk 1 above)
Comment (miranov25) [reply Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx]: Storing integer will be indeed enough

Comment (shahor02) [reply Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx]: @matthias-kleiner : the difference is that to read it back you will need the class library loaded while the integer can be read with bare root.

**Review by shahor02 — APPROVED** [review]

**Pull request merged**